autometa/common/mag.py

```diff
@@ -63,28 +65,53 @@ def __init__(self, assembly, contigs, outdir=None):
         self.nseqs = len(self.contigs)
 
     @property
+    @lru_cache(maxsize=None)
+    def assembly_seqs(self):
+        with open(self.assembly) as fh:
+            return [seq for title,seq in SimpleFastaParser(fh)]
+
+    @property
+    @lru_cache(maxsize=None)
+    def mag_seqs(self):
```

Comment:
Is this function misnamed? I'm just wondering if you meant to call this `mg_seqs` because I'm not sure `mag_seqs` makes sense as a name for this.